photon-client/src/integTest/scala/com/linkedin/photon/ml/cli/game/training/GameTrainingDriverIntegTest.scala

```diff
@@ -131,6 +131,48 @@ class GameTrainingDriverIntegTest extends SparkTestUtils with GameTestUtils with
     assertTrue(AvroUtils.modelContainsIntercept(sc, bestFixedEffectModelPath))
   }
 
+  /**
+   * Test GAME training with a fixed effect model only and hyperparameter tuning. Note that the best model may not be
+   * one of the tuned models.
+   */
+  @Test
+  def testFixedEffectsWithHyperparameterTuning(): Unit =
+    sparkTest("testFixedEffectsWithHyperparameterTuning", useKryo = true) {
+
+      // This is a baseline RMSE capture from an assumed-correct implementation on 4/14/2016
+      val errorThreshold = 1.7
```

Comment:
errorThreshold is 2.37 for other tests. 1.7 is lower but i wonder which one is correct.